go.mod

```diff
@@ -39,3 +39,6 @@ replace github.com/cosmos/ibc-go => github.com/crypto-org-chain/ibc-go v1.0.1-ho
 replace github.com/peggyjv/gravity-bridge/module => github.com/crypto-org-chain/gravity-bridge/module v0.1.22-0.20210923075944-630e7235f4b2
 
 replace github.com/cosmos/iavl => github.com/cosmos/iavl v0.17.1
+
+// FIXME: update after PR merged: https://github.com/tharsis/ethermint/pull/590
+replace github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.4.2-0.20210924142240-9589b2fa9f78
```

Comment:
lets wait the PR to be merge so that we can use the official repo

Reply:
done